### PR TITLE
new jira support link

### PR DESCRIPTION
**Pull request by dkhasham**
related Ticket: https://synjira.atlassian.net/browse/CELDEV-533
??https://synjira.atlassian.net/browse/AIPD-326

### CODE REVIEW

File: src/main/webapp/resources/celJS/initCelements.js [hunk 1]
```diff
@@ -818,4 +818,32 @@
...
+    jQuery(".celements_menu_bar_support_item").click(function(e) {
+      e.preventDefault();
+      showCollectorDialog();
+    });
+  };
+
+  var openJiraSupportBoxInit = function() {
+    // Requires jQuery!
+    jQuery.ajax({
+        url: "https://synjira.atlassian.net/s/d41d8cd98f00b204e9800998ecf8427e-T/-qxrqfl/b/c/3d70dff4c40bd20e976d5936642e2171/_/download/batch/com.atlassian.jira.collector.plugin.jira-issue-collector-plugin:issuecollector-embededjs/com.atlassian.jira.collector.plugin.jira-issue-collector-plugin:issuecollector-embededjs.js?locale=en-UK&collectorId=3195cde3",
```
Comment (fpichler) [inline src/main/webapp/resources/celJS/initCelements.js]: this URL is possibly not suited for an open source project. Please move to configuration file.

File: src/main/webapp/resources/celJS/adminUi/menubar.js [hunk 2]
```diff
@@ -207,19 +207,10 @@ var initMenuBarListener = function() {
...
   if (typeof my_menu_bar != 'undefined') {
     $$("body")[0].observe('click', my_menu_bar.show_menu_item.bind(my_menu_bar));
   }
 };
 
 Event.observe(window, 'load', initMenuBarListener);
-
-var openZendeskHandler = function(event) {
-  event.stop();
-  Cel_Zenbox.show();
```
Comment (fpichler) [inline src/main/webapp/resources/celJS/adminUi/menubar.js:224]: where is Cel_Zendbox defined? please remove the old code.

**Review by fpichler — CHANGES_REQUESTED** [review]

File: src/main/webapp/resources/celJS/adminUi/menubar.js (hunk 2 above)
Comment (dkhasham) [reply src/main/webapp/resources/celJS/adminUi/menubar.js:224]: old code removed

File: src/main/webapp/resources/celJS/supportLink/supportLink.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    jQuery(".celements_menu_bar_support_item").click(function(e) {
+      e.preventDefault();
+      showCollectorDialog();
+    });
+  };
+
+  var openJiraSupportBoxInit = function() {
+    // Requires jQuery!
+    jQuery.ajax({
+        url: Validation.messages.get("supportLinkURL"),
```
Comment (fpichler) [inline src/main/webapp/resources/celJS/supportLink/supportLink.js]: please use window.celMessages not Validation messages. Validation is only loaded on a Form-Page.

File: src/main/webapp/templates/celMacros/CelValidationMessages.vm [hunk 4]
```diff
@@ -17,3 +17,4 @@ $jsonBuilder.addStringProperty('validate-selection', 'Please make a selection')
 $jsonBuilder.addStringProperty('validate-one-required', 'Please select one of the above options.')
 $jsonBuilder.addStringProperty('validate-email-equal', "$!adminMsg.get('cel_form_valid_email-equal')")
 $jsonBuilder.addStringProperty('validate-docname', "$adminMsg.get('cel_tm_validate_warning')")
+$jsonBuilder.addStringProperty('supportLinkURL', "$adminMsg.get('cel_supportLink_url')")
```
Comment (fpichler) [inline src/main/webapp/templates/celMacros/CelValidationMessages.vm]: move to celMacros/CelMessages.vm

**Review by fpichler — CHANGES_REQUESTED** [review]

**Comment by fpichler** [comment]
please change to CELDEV-533

**Comment by fpichler** [comment]
duplicate to 55 please remove branch and close PR

**Pull request closed without merging**